framework_tvb/tvb/adapters/exporters/export_manager.py

```diff
@@ -253,6 +256,18 @@ def export_simulator_configuration(self, burst_id):
             raise InvalidExportDataException("Could not find burst with ID " + str(burst_id))
 
         op_folder = FilesHelper().get_project_folder(burst.project, str(burst.fk_simulation))
+        tmp_folder = os.path.join(TvbProfile.current.TVB_TEMP_FOLDER, self.EXPORTED_SIMULATION_NAME)
+
+        if not os.path.exists(tmp_folder):
+            os.makedirs(tmp_folder)
+        for r, d, files in os.walk(op_folder):
+            for file in files:
+                dest = os.path.join(tmp_folder, file)
+                initial_path = os.path.join(r, file)
+                FilesHelper().copy_file(initial_path, dest)
+                dest_file_class = H5File.determine_type(dest)
+                if dest_file_class is SimulatorAdapterModel:
+                    H5File.remove_metadata_param(dest, 'history_gid')
```

Comment:
Instead of reading the type from each H5 file, we could use `BurstConfiguration.simulator_gid` to directly determine the SimulatorAdapterModel.h5 file with:` h5.determine_filepath(sim_gid, tmp_dir)`